srcpkgs/docker-cli/update

```diff
@@ -0,0 +1 @@
+pattern='href="/docker/cli/releases/tag/v\K[\d\.]+(?=")'
```

Comment:
This is probably overkill. Why not just
```shell
pattern="v\K[\d\.]+(?=\.tar\.gz)"
```

Reply:
I actually haven't updated this yet, I forgot to not include it.